specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/examples/galleryExamples/GalleryApplication_Delete.json

```diff
@@ -7,8 +7,23 @@
     "galleryApplicationName": "myGalleryApplicationName"
   },
   "responses": {
-    "200": {},
-    "202": {},
-    "204": {}
+    "200": {
+      "headers": {
+        "Azure-AsyncOperation": "https://management.azure.com/subscriptions/000000-8de3-42e7-b00e-8970003527749a/providers/Microsoft.Compute/locations/eastus/capsOperations/0012b61c-2d36-40bc-b7ed-1f0e48757277?api-version=2021-07-01",
+        "Location": "https://management.azure.com/subscriptions/000000-8de3-42e7-b00e-8970003527749a/providers/Microsoft.Compute/locations/eastus/capsOperations/0012b61c-2d36-40bc-b7ed-1f0e48757277?monitor=true&api-version=2021-07-01"
```

Comment:
If Location is in fact returned by these API calls, I could not see that, then this is fine. Please ensure that is the case. If it is not, then the backend code will need to change to do so,